install/ansible/tasks/bootstrap_centos.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "Install required packages on CentOS "
+  yum:
+    name: "{{ item }}"
+    state: latest
+    with_items:
+      - unzip
```

Comment:
Why do we need `unzip`?

Reply:
for the ansible tar , unzip needs to be installed

Reply:
Good to know.